client/src/components/modal-forms/EditRosStartDate.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
```

Comment:
I feel like these three components **could** be cleaned to a common component, but may not be worth the effort

Reply:
I'll need to look into it either way. Otherwise, `SonarCloud` checks aren't going to pass 😅